xarray/test/test_dataset.py

```diff
@@ -2494,3 +2494,36 @@ def test_setattr_raises(self):
             ds.foo = 2
         with self.assertRaisesRegexp(AttributeError, 'cannot set attr'):
             ds.other = 2
+
+    def test_get_variables_by_attributes(self):
+        ds = Dataset({'temperature_0': (['t'],  [0], dict(standard_name='air_potential_temperature', height='0 m')),
+                      'temperature_10': (['t'],  [0], dict(standard_name='air_potential_temperature', height='10 m')),
+                      'precipitation': (['t'], [0], dict(standard_name='convective_precipitation_flux'))},
+                    coords={'time': (['t'], [0], dict(axis='T'))})
+
+        # Test return empty Dataset.
+        ds.get_variables_by_attributes(standard_name='invalid_standard_name')
+        new_ds = ds.get_variables_by_attributes(standard_name='invalid_standard_name')
+        self.assertFalse(bool(new_ds))
```

Comment:
@MaximilianR is this OK to test for am empty `Dataset`? Pandas has the `.empty` property, but I could not find am equivalent on `xarray`.